tabs/pid_tuning.js

```diff
@@ -970,13 +1062,17 @@ TABS.pid_tuning.initRatesPreview = function () {
     this.model = new Model($('.rates_preview'), $('.rates_preview canvas'));
 
     $(window).on('resize', $.proxy(this.model.resize, this.model));
+    $(window).on('resize', $.proxy(this.updateRatesLabels, this));
 };
 
 TABS.pid_tuning.renderModel = function () {
     if (this.keepRendering) { requestAnimationFrame(this.renderModel.bind(this)); }
 
     if (!this.clock) { this.clock = new THREE.Clock(); }
 
+    if (!this.oldRC) {this.oldRC = [RC.channels[0], RC.channels[1], RC.channels[2]];}
+    if (this.updateRequired==null) this.updateRequired = new Object(false);
```

Comment:
Please use blocks even for single statements. This makes the code a lot more maintainable.

Also, why are you doing this? `updateRequired` is never set to null, but even if it was, null is falsey, so the check is unneeded. In general, it is good practice to initialise object members at the very top of the file: https://github.com/GaryKeeble/betaflight-configurator/blob/3a77c7fadea9c0d73696726cd3820378ee10a819/tabs/pid_tuning.js#L4